src/plugins/smith_waterman/src/sw_cuda_cpp.cpp

```diff
@@ -202,8 +202,8 @@ QList<resType> calculateOnGPU(const char *seqLib, int seqLibLength, ScoreType *q
     cudaMalloc((void **)&g_directionsRec, sizeQ);
 
     if (U2::SmithWatermanSettings::MULTIPLE_ALIGNMENT == resultView) {
-        cudaError errorMatrix = cudaMalloc(reinterpret_cast<void **>(&g_directionsMatrix), directionMatrixSize);
```

Comment:
Please do not remove, but add error checks to all cudaMallocs in this file
Otherwise it will be a crash.